AutoMapper.OData.EFCore.Tests/GetQueryTests.cs

```diff
@@ -556,7 +556,7 @@ public async void FilteringOnRoot_ChildCollection_AndChildCollectionOfChildColle
             (
                 await Get<CategoryModel, Category>
                 (
-                    "/CategoryModel?$top=5&$expand=Products($filter=SupplierAddress/City ne '';$orderby=ProductName;$expand=AlternateAddresses($filter=City ne '';$orderby=City desc),SupplierAddress)&$filter=CategoryName ne ''&$orderby=CategoryName asc",
+                    "/CategoryModel?$top=5&$expand=Products($filter=SupplierAddress/City ne '';$orderby=ProductName;$expand=AlternateAddresses($filter=City ne '';$orderby=City desc;$top=1),SupplierAddress)&$filter=CategoryName ne ''&$orderby=CategoryName asc",
```

Comment:
This change works with or without the fix in ProjectionVisitor.  Maybe add a separate test with a case-insensitive URL?

Reply:
Actually, the URL casing doesn't trigger this problem as the parser seem to normalize them to the property names in the EDM so they work in any casing.

Enabling lower camel case changes the name of the properties in the EDM, which then triggers this error. I thought we could simply add this to the query tests as I had not found a test case where $top or $skip was called, which seem to now be added.

I could certainly add a new test (to the web tests only) but need to enable the lower camel case by default. Would that be ok?

Reply:
Not sure what you mean by enabling by default, but you're saying that the configuration and not the case of the URL itself causes the error, then multiple existing test will fail if we enable  lower camel case. The PR should be just two lines of code in that case.

- The fix.
- Add `builder.EnableLowerCamelCase();` [here](https://github.com/hasandogu/AutoMapper.Extensions.OData/blob/a8f6795ba358bec334f0c7b186bd6b6aed47eb1c/AutoMapper.OData.EFCore.Tests/GetTests.cs#L436) (so tests fail without the fix).

Reply:
We should already have a test with the $take for child collections.   Your fork is a few commits behind.